examples/issue/components/IssueEditor.js

```diff
@@ -36,7 +36,7 @@ export default class IssueEditor extends React.Component {
     };
   }
 
-  onChange = (editorState, cb) => this.setState({ editorState }, cb);
+  onChange = (editorState) => this.setState({ editorState });
```

Comment:
Oh I see what you are saying, hm it might be have been an artifact from when I pulled it out